jps-plugin/src/org/intellij/erlang/jps/builder/ErlangBuilder.java

```diff
@@ -266,27 +286,28 @@ private static void addIncludePaths(@NotNull GeneralCommandLine commandLine, @Nu
   @NotNull
   private static List<String> getErlangModulePaths(@NotNull ErlangTarget target,
                                                    @NotNull CompileContext context,
-                                                   @NotNull List<String> dirtyFilePaths,
                                                    boolean isTest) {
-    List<ErlangFileDescriptor> fileDescriptors = getErlangFileDescriptorFromConfig(target, context, isTest);
-    return fileDescriptors != null ?
-           getSortedErlangModulePathsToCompile(fileDescriptors, dirtyFilePaths) :
-           getErlangModulePathsDefault(target, isTest);
+    List<String> erlangBuilderDirtyFilesInfo = getErlangModulePathsFromTarget(target, context, isTest);
```

Comment:
Look at this variable name:
1) "erlangBuilder" is the name of your class - no value
2) "DirtyFiles" - yes, these indeed are dirty files, but these are not just files - these are modules
3) "Info" - no value at all